jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
+
+export GEM_SOURCE=http://rubygems.delivery.puppetlabs.net
+
+export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-centos6-64ma}"
+export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-beaker/install/foss/}"
+export BEAKER_CONFIG="scripts/hosts.cfg"
+export BEAKER_KEYFILE="~/.ssh/id_rsa-acceptance"
+export BEAKER_HELPER="beaker/helper.rb"
+export BEAKER_INSTALL_TYPE="foss"
```

Comment:
Hmm...I'm wondering if this is the correct value to use here.  I'm seeing `aio` used now.  Not sure if we should change this though, but if we run into pathing problems we should keep this in mind.